packages/kbn-unified-field-list/src/components/field_stats/field_stats.tsx

```diff
@@ -7,13 +7,9 @@
  */
 
 import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react';
-import {
-  type DataView,
-  type DataViewField,
-  ES_FIELD_TYPES,
-  getEsQueryConfig,
-  KBN_FIELD_TYPES,
-} from '@kbn/data-plugin/common';
+import type { DataView, DataViewField } from '@kbn/data-plugin/common';
+import { ES_FIELD_TYPES, KBN_FIELD_TYPES } from '@kbn/field-types';
+import { getEsQueryConfig } from '@kbn/data-service/src/es_query';
```

Comment:
It doesn't seem to be a requirement, but should we neithertheless prefer the imports to be without the `src/` path? :
```suggestion
import { getEsQueryConfig } from '@kbn/data-service';
```

Reply:
@kertal I noticed that using subpaths helped to reduce the bundle size.